src/workerd/server/workerd-api.c++

```diff
@@ -398,7 +398,8 @@ kj::Maybe<jsg::ModuleRegistry::ModuleInfo> WorkerdApi::tryCompileModule(jsg::Loc
               lock, Impl::compileJsonGlobal(lock, module.getJson())));
     }
     case config::Worker::Module::ES_MODULE: {
-      return jsg::ModuleRegistry::ModuleInfo(lock, module.getName(), module.getEsModule(),
+      // TODO(soon): Make sure passing `{}` to compile cache is desired.
+      return jsg::ModuleRegistry::ModuleInfo(lock, module.getName(), module.getEsModule(), {},
```

Comment:
maybe nullptr looks better in this context?

Also please `nullptr /* compile cache */,`

Reply:
Definitely!